src/components/FileInput.js

```diff
@@ -22,6 +22,7 @@ const FileInput = ({
   iconPosition,
   colspan,
   inputProps,
+  popupProps,
```

Comment:
Don't forget to include this in propTypes

Reply:
Done!